decidim-core/lib/tasks/procfile.rake

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require "thor"
+
+namespace :decidim do
+  namespace :procfile do
+    desc "Generates a dummy app for testing in external installations"
+    task :install do
+      actions :create_file, "Procfile.dev", <<~RUBY
+        web: bin/rails server -b 0.0.0.0 -p 3000
+        shakapacker: bin/webpacker-dev-server
+      RUBY
+
+      actions :create_file, "bin/dev", %{#!/usr/bin/env bash
+
+runner=foreman
+
+if command -v overmind &> /dev/null
+then
+  runner=overmind
+elif ! command -v foreman &> /dev/null
+then
+  echo "Installing foreman..."
+  gem install foreman
+fi
+
+$runner start -f Procfile.dev}
+
+      actions :chmod, "bin/dev", 777
```

Comment:
I think 777 is not good.
```suggestion
      actions :chmod, "bin/dev", 755
```

Rails default is also 755 for these kinds of files:
https://github.com/rails/jsbundling-rails/blob/b10503105a4ceb26358a39fffa0c5fd351bac3f7/lib/install/install.rb#L46